interface/src/app/tribe_auth/resource.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * The resources for fetching the OAuth2 token from the tribe_client url
+ */
+angular.module("adage.tribe_auth.resource", ['ngResource'])
+    .factory('User', ['$resource', function($resource) {
+        return $resource('/tribe_client/return_user', {}, {
+            query: {
+                method: 'GET',
+                isArray: false
+            }
+        });
+    }])
+
+;
```

Comment:
The way I think about this is that it's not actually a "line" of the file unless the line is terminated! It works fine, but only because the parser is not being overly strict.